apps/server/lib/lexical/server/code_intelligence/completion.ex

```diff
@@ -70,6 +78,16 @@ defmodule Lexical.Server.CodeIntelligence.Completion do
     end
   end
 
+  defp prefix_is_trigger?(env) do
```

Comment:
The `Completion.Context.trigger_kind` will be `:trigger_character` if completion was triggered by a trigger character.

Reply:
I don't quite understand this.

Reply:
If you want to figure out if the completion was caused by a trigger character (which is the name of this function), check the `Completion.Context.trigger_kind`. If the value is `trigger_character` then completion was caused by a trigger character.

Reply:
@scottming this is still unaddressed.

Reply:
> If you want to figure out if the completion was caused by a trigger character (which is the name of this function)

I think `trigger_character` is not the same as this function,  like if we print the `trigger_kind`, all of the tests of `  describe "in struct arguments" do` in `struct-fields` are `trigger_character`.

but what I want is: if the prefix is one of `[".", "@", "&", "%", "^", ":", "!", "-", "~"]`, then don't trigger the struct fields completions.